src/test/java/org/broadinstitute/hellbender/utils/SequenceDictionaryUtilsUnitTest.java

```diff
@@ -46,8 +73,10 @@
                 new SimpleInterval("chrM", 1, 1),
                 new SimpleInterval("chr1", 1, 1));
 
+        final List<SAMSequenceRecord> chr1_gh19 = Arrays.asList(CHR1_HG19);
```

Comment:
Or better, just delete it and use `Arrays.asList(CHR1_HG19)` in the one place this is used below.